test/fixtures/common/configuration.ts

```diff
@@ -59,7 +59,7 @@ export const generateFixtureBankTransferConfiguration = (): BankTransferConfigur
     virtualBankAccountsThreshold: 20,
     virtualBankAccountsFetchCount: 10,
 
-    expirationTimeShift: { enabled: true, value: new Date().toISOString() },
+    expirationTimeShift: { enabled: true, value: "09:00:00.000+09:00" },
```

Comment:
This does not change much as it is in the fixture but better having test data as close as possible from reality.